Examples/NewModuleTemplate/plasterManifest.xml

```diff
@@ -53,6 +53,10 @@
         </parameter>
     </parameters>
     <content>
+        <message>
+Scaffold a PowerShell Module with the files required to run Pester tests, build with PSake and publish to the PSGallery.
+
```

Comment:
Yep, the problems you list make sense.  Maybe if the developer wants extra newlines they can use PowerShell's newline syntax?  That way you can just trim all whitespace without trying to preserve the lines.  It makes the message text a little more messy but it makes the intent a lot clearer.  Something like this:

```xml
<message>
    Here's my first line.`r`n
    Here's my second line with two newlines following`r`n`r`n
    Here's my final line.
</message>
```